Makefile

```diff
@@ -77,7 +77,7 @@ fmt:
 	$(shell ./hack/update-gofmt.sh)
 
 container: hyperfed
-	cp -f $(HYPERFED_TARGET)-linux images/federation-v2/hyperfed
+	cp -f $(HYPERFED_TARGET)-$(HOST_OS) images/federation-v2/hyperfed
```

Comment:
@iamzhout thanks for your PR, but I think this change is not desirable as the container is based on linux. Please have a look at the Dockerfile [here](https://github.com/kubernetes-sigs/federation-v2/blob/master/images/federation-v2/Dockerfile)

Reply:
I get your point which I just missed and submitted previous quick fix.

I have updated the fix and verified locally on my macOS laptop, please check if this new fix make sense.

Thanks!

Reply:
@iamzhout thanks again. @yamt can you please verify the same, given you did the macos relevant changes. The fix seems right to me, as the container needs to always have the linux bin only.

Reply:
yes, it should always be a linux binary.